theme/assets/stylesheets/extra.61026d2ee9f4.css

```
@@ -42,11 +41,44 @@
+  .md-nav--lifted > .md-nav__list {
+    &:has(> .md-nav__item--sidebar-sticky-bottom) { padding-bottom: 1.2rem; }
+    > .md-nav__item--bosh-sidebar-sticky-bottom {
+        position: absolute;
+        bottom: 0;
+        width: 100%;
+        background: inherit;
+    }
+  }
+}
+
+@media screen and (min-width: 76.25em) {
```

Stylelint (17.14.0)

[error] Unclosed block (CssSyntaxError)

(CssSyntaxError)